src/jitlayers.cpp

```diff
@@ -305,28 +312,8 @@ void JuliaOJIT::DebugObjectRegistrar::registerObject(RTDyldObjHandleT H, const O
     const ObjT& Object = Obj;
 #endif
 
-    object::OwningBinary<object::ObjectFile> SavedObject = LO->getObjectForDebug(*Object);
```

Comment:
Do we no longer need to do this? Or is it incompatible with LLVM10?

Reply:
LLVM doesn't want us to do this, and has been making it increasingly difficult to avoid making these several copies. We used to have private copies of the llvm::DebugObjectRegistrar code to avoid these copies, but have since removed it (so this code served no real purpose). Unless we planning to convince LLVM to use `std::unique_ptr` less, the current strategy for C++ manual memory management now seems to be just to make private copies of everything. The things we do to avoid having a garbage collector...